src/preact.d.ts

```diff
@@ -5,7 +5,9 @@ declare namespace preact {
 	}
 
 	interface PreactHTMLAttributes {
+		dangerouslySetInnerHTML?:Object;
```

Comment:
Actually, not sure if this should be in PreactHTMLAttributes or just HTMLAttributes